### PR TITLE
Update File service API client maintenance message to read 'Uploads are temporarily disabled due to platform maintenance. Please try again in a few minutes.'

### PR DESCRIPTION
**Pull request by rinatkhaziev**
## Description

A minor wording update to read `Uploads are temporarily disabled due to Files service maintenance. Please try again in a few minutes.` instead of `Uploads are temporarily disabled due to Files service maintenance. Please try again later.`

## Checklist

Please make sure the items below have been covered before requesting a review:

- [ ] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [ ] I've created a changelog description that aligns with the provided examples.

### CODE REVIEW

**Review by parkcityj — APPROVED** [review]

File: files/class-api-client.php [hunk 1]
```diff
@@ -304,7 +304,7 @@ public function get_unique_filename( $file_path ) {
 		if ( 503 === $response_code ) {
 			return new WP_Error(
 				'file-service-readonly',
-				__( 'Uploads are temporarily disabled due to Files service maintenance. Please try again later.' )
+				__( 'Uploads are temporarily disabled due to Files service maintenance. Please try again in a few minutes.' )
```
Comment (mjangda) [inline files/class-api-client.php:307]:
```suggestion
				__( 'Uploads are temporarily disabled due to platform maintenance. Please try again in a few minutes.' )
```

**Review by mjangda — COMMENTED** [review]
Minor wording change

**Review by rebeccahum — APPROVED** [review]

**Review by WPprodigy — APPROVED** [review]

**Review by rebeccahum — APPROVED** [review]

**Review by jesse-greathouse — APPROVED** [review]
LGTM

**Comment by rinatkhaziev** [comment]
r2196-stacks

**Pull request merged**